Makefile

```diff
@@ -2,112 +2,108 @@ PROJECT_ROOT		:= github.com/infobloxopen/atlas-contacts-app
 BUILD_PATH  		:= bin
 DOCKERFILE_PATH		:= $(CURDIR)/docker
 
-IMAGE_VERSION		?= latest
+# configuration for image names
+USERNAME        := $(USER)
+GIT_COMMIT      := $(shell git describe --dirty=-unsupported --always || echo pre-commit)
+IMAGE_VERSION   ?= $(USERNAME)-dev-$(GIT_COMMIT)
+IMAGE_REGISTRY  ?= infoblox
 
+# configuration for server binary and image
 SERVER_BINARY 		:= $(BUILD_PATH)/server
-SERVER_PATH 		:= $(PROJECT_ROOT)/cmd/server
-SERVER_IMAGE		:= infoblox/contacts-server:$(IMAGE_VERSION)
-SERVER_DOCKERFILE 	:= $(DOCKERFILE_PATH)/Dockerfile.contacts-server
-
-GATEWAY_BINARY 		:= $(BUILD_PATH)/gateway
-GATEWAY_PATH		:= $(PROJECT_ROOT)/cmd/gateway
-GATEWAY_IMAGE		:= infoblox/contacts-gateway:$(IMAGE_VERSION)
-GATEWAY_DOCKERFILE 	:= $(DOCKERFILE_PATH)/Dockerfile.contacts-gateway
-
-GO_PATH              	:= /go
-SRCROOT_ON_HOST      	:= $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
-SRCROOT_IN_CONTAINER	:= $(GO_PATH)/src/$(PROJECT_ROOT)
-GO_CACHE             	:= -pkgdir $(SRCROOT_IN_CONTAINER)/$(BUILD_PATH)/go-cache
-
+SERVER_PATH 			:= $(PROJECT_ROOT)/cmd/server
+SERVER_IMAGE			:= $(IMAGE_REGISTRY)/contacts
+SERVER_DOCKERFILE := $(DOCKERFILE_PATH)/Dockerfile.server
+
+# configuration for gateway binary and image
+GATEWAY_BINARY 	   := $(BUILD_PATH)/gateway
+GATEWAY_PATH		   := $(PROJECT_ROOT)/cmd/gateway
+GATEWAY_IMAGE		   := $(IMAGE_REGISTRY)/contacts-gateway
+GATEWAY_DOCKERFILE := $(DOCKERFILE_PATH)/Dockerfile.gateway
+
+# configuration for the protobuf gentool
+SRCROOT_ON_HOST		:= $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+SRCROOT_IN_CONTAINER	:= /go/src/$(PROJECT_ROOT)
 DOCKER_RUNNER        	:= docker run --rm
 DOCKER_RUNNER        	+= -v $(SRCROOT_ON_HOST):$(SRCROOT_IN_CONTAINER)
-DOCKER_BUILDER       	:= infoblox/buildtool:v8
-DOCKER_GENERATOR     	:= infoblox/atlas-gentool:v2
-GENERATOR            	:= $(DOCKER_RUNNER) $(DOCKER_GENERATOR)
+DOCKER_GENERATOR	:= infoblox/atlas-gentool:v2
+GENERATOR		:= $(DOCKER_RUNNER) $(DOCKER_GENERATOR)
 
-BUILD_TYPE ?= "default"
-ifeq ($(BUILD_TYPE), "default")
-	BUILDER        := $(DOCKER_RUNNER) -w $(SRCROOT_IN_CONTAINER) $(DOCKER_BUILDER)
-endif
+# configuration for the database
+DATBASE_ADDRESS		:= localhost:5432
```

Comment:
Typo here?

Reply:
good catch! I think it's supposed to be `DATBASS`. Caleb is all about dat bass... #notreble

Reply:
Actually, this is propagated from the bootstrap output, raised it with Zach. He confirmed Dan's suspicions about the intended spelling.